### ISSUE TITLE
[BUG] Navigation Bar overlaying mobile menu/sidebar

### ISSUE THREAD

**Issue by caioalexandrebr** [Gympass/yoga#343]
**Describe the bug**
Currently the mobile menu/sidebar is cutting the presentation of items positioned at the bottom of the list. Chrome NavigationBar overlaps the end of the listing.

https://user-images.githubusercontent.com/31045534/134536445-c2d2a72a-7f3b-49d9-9f09-b2de3dd175d7.mp4

**To Reproduce**
1. Go to https://gympass.github.io/yoga/components/getting-started/
2. Scroll to the bottom of the page.
3. In the menu mobile/sidebar, the last items listed are off the screen.

**Expected behavior**
It will be possible to view all items listed in the menu mobile/sidebar.

**Smartphone:**
 - iPhone 11
 - OS: 14.7
 - Chrome

**Comment by coelhucas** [comment]
Are you willing to pick up this issue?

**Comment by ggdaltoso** [comment]
Seems to be an easy fix. You just need to update a CSS unit from `vh` to `%` [here](https://github.com/Gympass/yoga/blob/master/packages/doc/src/components/Navigation/Navigation.jsx#L40)

```diff
-      height: calc(100vh - 70px);
+      height: calc(100% - 70px);
```

**Comment by caioalexandrebr** [comment]
Hi guys, thank you so much for your collaboration! 🚀 

I would love to fix this problem.

ty @coelhucas @ggdaltoso ❤️

**Comment by coelhucas** [comment]
As we discussed internally, it's being addressed in #356.

**Comment by coelhucas** [comment]
Fixed in #356 

| iOS | Android | Desktop |
| -- | -- | -- |
| ![image0](https://user-images.githubusercontent.com/28108272/136259115-7aae4848-33e4-4fe2-a0c9-f5a217af5e9d.png) | ![Screenshot_20211006-150331_Firefox Nightly](https://user-images.githubusercontent.com/28108272/136259137-b230f82b-63c5-47b4-926e-3decbc17a0eb.jpg) | ![image](https://user-images.githubusercontent.com/28108272/136259479-74974e5b-e675-49ea-b14d-f40ca457ae55.png) |


An interesting behaviour that I noticed, though, is that the order changed in my iPhone 🤔. But in my tablet, desktop and @caioalexandrebr's iPhone it keeps the same order. cc/ @ggdaltoso

**Comment by coelhucas** [comment]
Closing this since issue is solved. We'll evaluate the need of opening a new issue regarding this sorting order that's inverted in Firefox. Feel free to reopen in case of any doubts or other problems.
